src/app/[locale]/client-filters.tsx

```diff
@@ -33,6 +43,7 @@ export default function ClientFilters({
   const [query, setQuery] = useState('');
   const [offset, setOffset] = useState(0);
   const [sort, setSort] = useState<Sort>(defaultSort);
+  const [mobileFiltersOpen, setMobileFiltersOpen] = useState(false);
```

Comment:
About the name 'mobile': is this about a device (e.g. a telephone) or about a function that provides 'mobile filters'?

Reply:
It's about the filter sidebar on mobile devices. The boolean is true if the sidebar is visible. I will rename it to `showFilterSidebarOnMobile`

![mobileFiltersOpen](https://user-images.githubusercontent.com/1481602/226353389-e4ce5f57-047b-43c6-92f9-7fcc2f7560c4.gif)